### PR TITLE
adding hotspot integration testing, reworkig maker section

### PR DESCRIPTION
**Pull request by PharkMillups**
* Adds hotspot integration guide
* adds Maker section
* reorders some pages

### CODE REVIEW

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+
+# Hotspot Integration Testing
+
+This guide is intended for companies that have been approved via HIP19, or are considering submitting an application. 
+
+## Helium Hotspot Wallet App Compatibility
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: Change to "Hotspot App Compatibility"

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+For optimal user experience, we recommend Hotspots be equipped with compatible bluetooth and Wi-Fi functionality. This allows a seamless onboarding experience.
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: capitalize Bluetooth

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+:::
+
+Once you’re ready to proceed, request a Maker account and replace the test payer address with your own maker address.
+
+
+## Requesting a Maker Account
+
+Once you are confident that the transaction is sound and well crafted, request a Maker Account from your DeWi or Helium contact. In your request, include the Maker Name and how many location asserts you will include for your customers. A minimum of `1` included location assertion is required. That value is denoted under `locationNonceLimit`.
+
+All Maker Accounts are public on the blockchain [and can be found here](https://helium-onboarding-server.herokuapp.com/api/v2/makers).
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: update to https://onboarding.dewi.org/api/v2/makers

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+### Testing
+
+When you’re ready to start integration with the Test Maker account with the mobile wallet app, send us details about your Test Hotspot in this JSON format:
+
+```json
+"id": 6488, 
+"mac_eth0": "dc:a6:32:d8:e7:5c", 
+"mac_wlan0": "dc:a6:32:d8:e7:5e", 
+"onboarding_address": "blah",
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: "onboarding_address": "retrieve this b58 address from miner",

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+
+
+## Adding Hotspots to the Onboarding Server
+
+### Testing
+
+When you’re ready to start integration with the Test Maker account with the mobile wallet app, send us details about your Test Hotspot in this JSON format:
+
+```json
+"id": 6488,
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: "id": any number,

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+"rpi_serial": "0000011031311847"
+```
+
+### Production
+
+When you move to production, the JSON should be formatted like this. Note the addition of the `batch` field:
+
+```json
+"batch": "batch_name",  
+"id": 6488,
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: "id": any number,

File: docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx [hunk 7]
```diff
@@ -0,0 +1,176 @@
...
+### Production
+
+When you move to production, the JSON should be formatted like this. Note the addition of the `batch` field:
+
+```json
+"batch": "batch_name",  
+"id": 6488, 
+"mac_eth0": "dc:a6:32:d8:e7:5c", 
+"mac_wlan0": "dc:a6:32:d8:e7:5e", 
+"onboarding_address": "blah",
```
Comment (cokes518) [inline docs/mine-hnt/hotspot-makers/hotspot-integration-testing.mdx]: "onboarding_address": "retrieve this b58 address from miner",

**Review by kent-williams — APPROVED** [review]

**Pull request merged**